shell/common/skia_event_tracer_impl.cc

```diff
@@ -183,22 +186,67 @@ class FlutterEventTracer : public SkEventTracer {
     trace_release_context(trace_context);
 
 #else   // defined(OS_FUCHSIA)
+    std::vector<const char*> arg_names;
+    std::vector<std::string> arg_values;
+    for (int i = 0; i < num_args; i++) {
+      arg_names.push_back(p_arg_names[i]);
+      const uint8_t arg_type = p_arg_types[i];
+      const uint64_t arg_value = p_arg_values[i];
+      switch (arg_type) {
+        case TRACE_VALUE_TYPE_BOOL: {
+          SkiaTraceValueUnion value = {.as_uint = p_arg_values[i]};
```

Comment:
I believe this is blowing up MSVC - can't use this syntax there.

... another reason to go to clang on windows :)

Reply:
See my comment on |SkiaTraceValueUnion| above.  We should just be able to just convert this with "!!".  It should also be kept in sync with the Fuchsia version.